rust-runtime/aws-smithy-types/src/date_time/format.rs

```diff
@@ -377,6 +377,11 @@ pub(crate) mod rfc3339 {
     // Timezones not supported:
     // Not OK: 1985-04-12T23:20:50-02:00
     pub(crate) fn parse(s: &str) -> Result<DateTime, DateTimeParseError> {
+        if !matches!(s.chars().last(), Some('Z')) {
+            return Err(DateTimeParseError::Invalid(
+                "invalid RFC-3339 date-time: timezone offset not supported".into(),
```

Comment:
This error message should probably be rephrased since date times with a timezone offset actually are valid RFC-3339 timestamps:
```suggestion
                "Smithy does not support timezone offsets in RFC-3339 date times".into(),
```

Reply:
You're right, thanks!